src/main/java/net/openhft/chronicle/wire/ValueOut.java

```diff
@@ -967,6 +967,12 @@ default WireOut writeString(CharSequence x) {
         return text(x);
     }
 
+    default WireOut writeInt(IntConverter intConverter, int i) {
+        StringBuilder sb = Wires.acquireStringBuilder();
```

Comment:
maybe use a different -> Wires.acquireStringBuilder() instance

Reply:
Wires.acquireStringBuilder() is no longer public however Wires.acquireBytes() is #393